src/components/cylc/table/Table.vue

```diff
@@ -180,7 +187,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
                 <td>
                   <div class="d-flex align-content-center flex-nowrap">
                     <div class="mr-1">
-                      <Job
+                      <job
```

Comment:
Why make this lowercase, out of curiosity?

Reply:
Sorry my OCD. I believe I did that because it was lowercase in... the tree component? But no functional need to change it here :grimacing:

Reply:
Actually I would like to change the lowercase instances to uppercase, as it makes them stand out from normal HTML tags!

Reply:
+1! Good idea, and vive la consistance! 😄